plugins/common/common.go

```diff
@@ -134,8 +153,213 @@ func GetAppImageRepo(appName string) string {
 func VerifyImage(image string) bool {
 	imageCmd := NewShellCmd(strings.Join([]string{"docker inspect", image}, " "))
 	imageCmd.ShowOutput = false
-	if imageCmd.Execute() {
-		return true
+	return imageCmd.Execute()
+}
+
+// ContainerIsRunning checks to see if a container is running
+func ContainerIsRunning(containerId string) bool {
+	b, err := sh.Command("docker", "inspect", "--format", "'{{.State.Running}}'", containerId).Output()
+	if err != nil {
+		return false
+	}
+	return string(b[:]) == "true"
```

Comment:
`ContainerIsRunning` always returns `false` because `b` is `"true\n"` instead of `"true"`.